doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,90 @@
+  ambassador is not otherwise an active member of the project itself.
+
+## Ambassadors nominations
+
+Any member of the Node.js organization may nominate an ambassador.
+The Technical Steering Committee (TSC) will review/approve ambassadors.
+
+In the initial phase of the program:
+
+* there will be a maximum of 4 ambassadors
+* the term will be for the calendar year. After a year ambassadors automatically
+  drop out of the ambassador list and must be re-nominated
+* ambassadors will be documented on the nodejs/ambassadors README.md
+
+The TSC may choose to remove an ambassador if things are not working
+out in terms of the content promoted and alignment with the project.
+
+Members of the Node.js organization can nominate ambassadors by opening
+an issue in: TBD.
+
+## Reviewing content
+
+Ambassadors are asked to share links to new content before asking
+for the content to be promoted. The goal is to provide the opportunity
+for collaborators to help improve content as opposed to a detailed
+nitpick review.
+
+This is done by opening an issue in the nodejs/ambassadors
+repository on which access is limited to members of the Node.js
+organization. The issue may include the link to existing published
+content or a draft of the content itself.
+
+Interested members of the Node.js organization may comment/make
+suggestions in the issue and the ambassadors are expected to
+consider those suggestions and update the content when they feel it
+is appropriate.
+
+Explicit approval is not required in the issue (ie if there is zero
+input that's ok), but objections should addressed before
+requesting promotion.
+
+## Expectations on Ambassadors
+
+Ambassadors are expected to engage both within the project and externally
+in a manner that is consistent with the projects
+[Code of Conduct](https://github.com/nodejs/admin/blob/main/CODE_OF_CONDUCT.md)
+as well as the additional expectations outlined in
+[MemberExpectations](https://github.com/nodejs/admin/blob/main/MemberExpectations.md)
+for a leadership position.
+
+## Promoting content
+
+We trust ambassadors to only request promotion of content that lines
+up with the messages and topics outlined, that this content does not overly
+market their employer or product/business, and that they have followed the review
+process. An ambassador can request promotion of content in the following ways:
+
+* Requesting that a tweet be retweeted by the Node.js X account.
+* Requesting that a LinkedIn post be reposted by the Node.js LinkedIn
+  account.
+* At mentioning the nodejs/ambassadors team to ask other ambassadors to
+  repost
+* Posting a link to the content in the "what's new" issue in nodejs/ambassadors
+  so that it goes out on the news feed.
+
+Requests for a retweet or LinkedIn repost will be done by the foundation
+staff without any need for validation based on the request coming from
+an ambassador. These requests can be made through the existing social channel
```

Comment:
The way this is written, it says that foundation staff will make the requests. Maybe this is slightly less confusing?

```suggestion
Foundation staff will retweet or LinkedIn repost
without any need for validation based on the request coming from
an ambassador. These requests can be made through the existing social channel
```